src/components/Global/BreadCrumbs.vue

```diff
@@ -78,18 +78,22 @@ export default defineComponent({
   background-color: $colorRedWispPink;
   @include mediumScreen {
     height: 40px;
-    padding: 12px 32px;
     font-size: 1rem;
   }
 }
 .breadcrumbs__parent,
 .breadcrumbs__current {
-  padding: 12px 16px;
+  padding: 12px 5px;
+  @include mediumScreen {
+    padding: 12px 16px;
+  }
 }
 
 .breadcrumbs__back {
+  padding: 0;
   border: 2px solid $colorRedWispPink;
```

Comment:
I don't think there's anything wrong with how you do it now but if you would like to look at sccs documentation https://sass-lang.com/documentation/variables
the variable names are declared as kebab case not camel case ( thats used for javascript ). I think it would be good for you to do it in the same way so your code is easier to read for other developers in your future projects. 💡